content/team/collaboration.md

```diff
@@ -0,0 +1,54 @@
+of our community. To know more about members with specific 
+roles, visit the <a href="../community_roles/">Community Roles</a> page.
+
+## GSoC/SoCIS Students
+A brief explanation on GSoC & SoCIS
+
+| Program  | Name     | Project  |
+|----------|----------|----------|
+| GSoC 2020  | Jordi Eguren Brown   | <a href="https://summerofcode.withgoogle.com/archive/2020/projects/5699342318436352/" target="_blank" rel="noopener nofollow">TARDIS Code reestructure with factory method</a>   |
+| GSoC 2020  | Ezequiel Passaro   | <a href="https://summerofcode.withgoogle.com/archive/2020/projects/5301481479077888/" target="_blank" rel="noopener nofollow">Comparing TARDIS spectra generated with different atomic data sources</a> |
+| GSoC 2020  | Jaladh Singhal  | <a href="https://summerofcode.withgoogle.com/archive/2020/projects/5967738750631936/" target="_blank" rel="noopener nofollow">Jupyter Notebook Widgets for TARDIS</a>  |
+| GSoC 2019  | Ezequiel Passaro   | <a href="https://summerofcode.withgoogle.com/archive/2019/projects/5344591031566336/" target="_blank" rel="noopener nofollow">Expansion of the TARDIS Atomic Database</a>   |
+| GSoC 2019  | Jaladh Singhal  | <a href="https://summerofcode.withgoogle.com/archive/2019/projects/4723622378209280/" target="_blank" rel="noopener nofollow">Auto-generate Filter Curves & Photometry</a>  |
+| GSoC 2017  | Vaibhav Gupta  | <a href="https://summerofcode.withgoogle.com/archive/2017/projects/6578975069437952/" target="_blank" rel="noopener nofollow">Reading Simulation from file</a>  |
+| GSoC 2017  | Stefan Lietzau  | <a href="https://summerofcode.withgoogle.com/archive/2017/projects/6243763743621120/" target="_blank" rel="noopener nofollow">New module for the creation of spectra for TARDIS</a>  |
+| SOCIS 2016 | Tomas Bylund  | No Report  |
+| GSoC 2016  | Karan Desai   | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/4796129849901056/" target="_blank" rel="noopener nofollow"> Extensive Test Suite for TARDIS<a>   |
+| GSoC 2016  | Mikhail Mishin   | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/6699840280985600/" target="_blank" rel="noopener nofollow">Carsus - TARDIS support package for creating atomic datasets.</a>   |
+| GSoC 2016  | Fotis Tsamis  | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/5656164301799424/" target="_blank" rel="noopener nofollow">Improvement of the TARDIS configuration system</a>  |
+| GSoC 2015  | Aoife Boyle   | <a href="https://www.google-melange.com/archive/gsoc/2015/orgs/tardis_sn/projects/aboyle.html" target="_blank" rel="noopener nofollow">Implementation of new features into the new Tardis plasma module framework.</a>   |
+| GSoC 2015  | Vytautas Jančauskas   | <a href="https://www.google-melange.com/archive/gsoc/2015/orgs/tardis_sn/projects/bucket_brigade.html" target="_blank" rel="noopener nofollow">TARDIS-SN Fitting Multiple Supernova Spectra at Once</a>   |
+| GSoC 2014  | Vytautas Jančauskas   | No Report   |
```

Comment:
It was under PSF - https://www.google-melange.com/archive/gsoc/2014/orgs/python/projects/bucket_brigade.html